.circleci/config.yml

```diff
@@ -21,7 +21,7 @@ restore_cache_step: &restore_cache_step
         #                              still be able to do it. In order to achieve this we can change the value of the
         #                              CACHE_EXPIRE_HASH in our CircleCI's repo settings. Please use the format
         #                              'YYYY-MM-DD'. This way a new push on the branch is not required.
-        - tox-cache-{{ .Environment.CIRCLE_JOB }}-{{ .Environment.CACHE_EXPIRE_HASH }}
+        - tox-cache-{{ .Environment.CIRCLE_JOB }}-{{ checksum "tox.ini" }}
```

Comment:
We need this below on L28 as well.

Just make sure no usage of `CACHE_EXPIRE_HASH` exists anywhere.